src/Theme.elm

```diff
@@ -85,3 +94,17 @@ threeColumn : Css.CalculatedLength
 -}
 threeColumn =
     calc (pct 33) minus (rem 2)
+
+
+{-| A div with known vertical margin
+-}
+verticalSpacing : Html msg
```

Comment:
If his is the only export that is an element, rather than a style, should we stick it in it's own section headed by `-- Elements` vs `-- Style` vs `--Global ?directives?`